astro/src/content/docs/get-started/use-cases/machine-to-machine.mdx

```diff
@@ -259,56 +143,17 @@ This code checks the following parts of the access token:
 
 You must further check the permissions from the validated token using code like this helper method:
 
-```javascript
-const jose = require('jose');
-
-function hasPermission(apiEntityId, expectedPermission) {
-  return (req, res, next) => {
-    const decodedToken = jose.decodeJwt(req.verifiedToken);
-    if (checkPermission(decodedToken, apiEntityId, expectedPermission)) {
-      return next();
-    }
-
-    res.status(403);
-    res.send({ error: `You do not have permissions to do this.` });
-  }
-}
-
-function checkPermission(json, uuid, permission) {
-  if (!json.permissions || typeof json.permissions !== 'object') {
-    return false;
-  }
-  
-  return json.permissions[uuid]?.includes(permission) || false;
-}
-
-module.exports = hasPermission;
-```
+<RemoteCode lang="javascript"
+            url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-machine-to-machine/refs/heads/main/apis/services/hasPermission.js"
+            title="Validating the access token in the API" 
+            />
```

Comment:
Should this be removed from the `hasPermission.js` file?

```
// if (permissions.some((permission) => decodedToken.permissions.includes(permission))) return next();
```
The line is commented out, but `permissions` isn't declared anywhere, so I'm not sure where it came from. If this commented out code should stay, another comment explaining it would be useful.